docs/src/main/asciidoc/stork-kubernetes.adoc

```diff
@@ -0,0 +1,479 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Getting Started with SmallRye Stork
+:extension-status: preview
+
+// Temporary until stork in in the BOM
```

Comment:
isn't stork in the bom?

Reply:
I don't know really how it works, I copied it from the `docs/src/main/asciidoc/stork.adoc`